<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="hotkeySettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>快捷键设置</string>
    </property>
    <widget class="QCheckBox" name="ctrlCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ctrl</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="altCheckBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Alt</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="shiftCheckBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Shift</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="keyInputButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>输入按键</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>51</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="keyDelaySpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoEnterCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>自动输入回车</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyDelayLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>按键延迟：</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyDelayMsLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>毫秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="showConfigCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>165</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>启动时显示设置</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="displayLanguageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>显示语言：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="displayLanguageComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
